analyzers/tests/SonarAnalyzer.UnitTest/Rules/CollectionEmptinessCheckingTest.cs

```diff
@@ -30,10 +30,25 @@ public class CollectionEmptinessCheckingTest
     {
         [TestMethod]
         public void CollectionEmptinessChecking_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\CollectionEmptinessChecking.cs", new CS.CollectionEmptinessChecking());
+            new VerifierBuilder()
+                .AddAnalyzer(() => new CS.CollectionEmptinessChecking())
+                .AddPaths("CollectionEmptinessChecking.cs")
```

Comment:
You can actually do this, because rule and the codefix share this. And we don't have any other complicated scenario here.
```
private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.CollectionEmptinessChecking>().AddPaths("CollectionEmptinessChecking.cs");

[TestMethod]
        public void CollectionEmptinessChecking_CS() =>
            builderCS.Verify();
```

Reply:
If you don't like the idea of sharing `AddPaths`, the `builderCS` defining the shared rule should be used.

Reply:
I would argue that you want to have these tests isolated; and therefore not share setup. But I've you prefer this, I can change it.